scipy/stats/_hypotests.py

```diff
@@ -130,3 +131,133 @@ def epps_singleton_2samp(x, y, t=(0.4, 0.8)):
     p = chi2.sf(w, r)
 
     return Epps_Singleton_2sampResult(w, p)
+
+
+def poisson_etest(k1, k2, n1, n2, diff=0, alternative='two-sided'):
```

Comment:
Name of the test: Maybe a more generic name like `poission_means_test` ? then one could add further approaches such as the C-test at a later stage using a new keyword `method` that can be added without breaking backwards compatability

Reply:
also, to align with `ttest_ind_from_stats`, one could replace `k1` by `mean1` (and then compute k1 in the code), `n1` by `nobs1`. or do you think that using the count is better?

Reply:
what about `poisson_test_from_stats` or `poisson_means_test_from_stats`? to follow how the t-test family functions are named. though the last alternative maybe too long?

yes, i think using the `count` would be more straightforward. As the user would only need to sum all cases compared to the mean which also need to calculate `count` divided by `n`

Reply:
Well if we aim to generalize the name of test maybe make name more expressive would be the best. Let's settle for `poisson_means_test` since there is no wrapper, unlike the t-test